test/functional/interface_zmq_dash.py

```diff
@@ -70,7 +70,7 @@ def run_test(self):
             self.nodes[0].spork("SPORK_19_CHAINLOCKS_ENABLED", 0)
             self.wait_for_sporks_same()
             force_finish_mnsync(self.nodes[0])
-            # Create a LLMQ for testing
+            # Create an LLMQ for testing
```

Comment:
The general rule is to say it, and if it sounds wrong to change it :D saying `LLMQ` the first sound sounds like 'el' an E sound, so since it starts with a vowel sound, it should be an